assets/js/common/SuseManagerSettingsDialog/SuseManagerSettingsModal.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { noop } from 'lodash';
+import { format } from 'date-fns';
+import { EOS_LOCK_OUTLINED } from 'eos-icons-react';
+
+import Button from '@common/Button';
+import Modal from '@common/Modal';
+import Input, { Password, Textarea } from '@common/Input';
+import Label from '@common/Label';
+
+import { hasError } from './errors';
+
+const defaultErrors = [];
+
+function SuseManagerSettingsModal({
+  open = false,
+  loading = false,
+  initialUsername,
+  initialUrl,
+  certUploadDate,
+  errors = defaultErrors,
+  onSave = noop,
+  onCancel = noop,
+  onClearErrors = noop,
+}) {
+  const settingsExist = Boolean(certUploadDate);
+
+  const [url, setUrl] = useState(initialUrl);
+  const [username, setUsername] = useState(initialUsername);
+  const [password, setPassword] = useState('');
+  const [certificate, setCertificate] = useState('');
+  const [editingCertificate, setEditingCertificate] = useState(!settingsExist);
+  const [editingPassword, setEditingPassword] = useState(!settingsExist);
+
+  return (
+    <Modal title="Enter SUSE Manager Settings" open={open} onClose={onCancel}>
+      <div className="grid grid-cols-6 items-center my-5 gap-6">
+        <Label className="col-span-2" required>
+          SUSE Manager URL
+        </Label>
+        <Input
+          className="col-span-4"
+          value={url}
+          placeholder="Enter a URL"
+          error={hasError('url', errors)}
+          onChange={({ target: { value } }) => {
+            setUrl(value);
+            onClearErrors();
+          }}
+        />
+        <Label
+          className="col-span-2 self-start"
+          info="Only required for self-signed certificates"
+        >
+          CA Certificate
+        </Label>
+        {editingCertificate ? (
+          <Textarea
+            className="col-span-4"
+            value={certificate}
+            placeholder="Starts with -----BEGIN CERTIFICATE-----"
+            error={hasError('ca_cert', errors)}
+            onChange={({ target: { value } }) => {
+              setCertificate(value);
+              onClearErrors();
+            }}
+          />
+        ) : (
+          <div className="col-span-4 flex flex-row items-center justify-start p-5 border border-gray-200 rounded-md">
+            <EOS_LOCK_OUTLINED className="mr-3" size="25" />
```

Comment:
I guess we could/should reuse `CertificateUploadDate` component here. Or an adjustment of it to make it reusable here.